mlir/include/mlir/Dialect/SPIRV/IR/SPIRVBase.td

```diff
@@ -4146,7 +4146,12 @@ def SPIRV_Int32 : TypeAlias<I32, "Int32">;
 def SPIRV_Float32 : TypeAlias<F32, "Float32">;
 def SPIRV_Float : FloatOfWidths<[16, 32, 64]>;
 def SPIRV_Float16or32 : FloatOfWidths<[16, 32]>;
-def SPIRV_Vector : VectorOfLengthAndType<[2, 3, 4, 8, 16],
+// Remove the vector size restriction.
```

Comment:
This "Remove the vector size restriction." sentense does not need to be in the comment--it's the goal of this patch; but not a proper doc of `SPIRV_Vector`.